lua/entities/ent_jack_gmod_ezblackhole.lua

```diff
@@ -6,8 +6,10 @@ ENT.Category="JMod - EZ"
 ENT.Information="glhfggwpezpznore"
 ENT.PrintName="EZ Micro Black Hole"
 ENT.Spawnable=false
-ENT.AdminSpawnable=false
+ENT.AdminSpawnable=true
```

Comment:
i don't want the black hole to show up in the spawn menu in any case, it's a point entity

Reply:
Good catch, though it shouldn't affect anything. Amended.